tests/unit/name_tests.cpp

```diff
@@ -14,24 +14,28 @@ using std::string;
 
 using eosio::name;
 
-static constexpr uint64_t u64min = numeric_limits<uint64_t>::min(); // 0ULL
-static constexpr uint64_t u64max = numeric_limits<uint64_t>::max(); // 18446744073709551615ULL
+constexpr uint64_t u64max = numeric_limits<uint64_t>::max(); // 18446744073709551615ULL
 
 // Definitions in `eosio.cdt/libraries/eosio/name.hpp`
-EOSIO_TEST_BEGIN(name_type_test)
+EOSIO_TEST_BEGIN(name_type_test_ctr_num)
    //// constexpr name()
    CHECK_EQUAL( name{}.value, 0ULL )
 
    //// constexpr explicit name(uint64_t)
    CHECK_EQUAL( name{0ULL}.value, 0ULL )
    CHECK_EQUAL( name{1ULL}.value, 1ULL )
    CHECK_EQUAL( name{u64max}.value, u64max )
-
    //// constexpr explicit name(name::raw)
    CHECK_EQUAL( name{name::raw{0ULL}}.value, 0ULL )
    CHECK_EQUAL( name{name::raw{1ULL}}.value, 1ULL )
    CHECK_EQUAL( name{name::raw{u64max}}.value, u64max )
 
+   // test that constexpr constructor is evaluated at compile time
+   static_assert(name{0ULL}.value == 0ULL);   
```

Comment:
One minor format issue: either use one space after bracket or not, so that the style is consistent within the whole file.